examples/custom_context.py

```diff
@@ -0,0 +1,43 @@
+import discord
+from discord.ext import commands
+
+
+class MyContext(commands.Context):
+    async def tick(self, value):
+        # reacts to the message with an emoji
+        # depending on whether value is True or False
+        # if its True, itll add a green check mark
+        # otherwise, itll add a red cross mark
+        emoji = '\N{white heavy check mark}' if value else '\N{cross mark}'
+        try:
+            # this will react to the command author's message
+            await self.message.add_reaction(emoji)
+        except discord.HTTPException:
+            # sometimes errors occur during this, for example
+            # maybe you dont have permission to do that
+            # we dont mind, so we can just ignore them
+            pass
+
+
+class MyBot(commands.Bot):
+    async def get_context(self, message, *, cls=MyContext):
+        # when you override this method, you pass your new Context
+        # subclass to the super() method, which tells the bot to
+        # use the new MyContext class
+        return await super().get_context(message, cls=cls)
+        
+
+bot = MyBot(command_prefix='!')
+
+@bot.command()
+async def guess(ctx, number: int):
+    """ Guess a random number from 1 to 6. """
+    # explained in a previous example, this gives you
+    # a random number from 1-6
+    value = random.randint(1, 6)
+    # with your new helper function, you can add a
+    # green check mark if the guess was correct,
+    # or a red cross mark if it wasnt
+    await ctx.tick(number == value)
+    
+bot.run("your token here")
```

Comment:
Probably needs a comment saying that a token is like a password and shouldn't be hardcoded or shared. Maybe suggest a file etc.